beaconclient/prod_beacon_instance.go

```diff
@@ -21,6 +21,9 @@ type ProdBeaconInstance struct {
 	// feature flags
 	ffUseV1PublishBlockEndpoint  bool
 	ffUseSSZEncodingPublishBlock bool
+
+	// http clients
+	publishingClient *http.Client
```

Comment:
**question:**

how is this client initialised?  as in, where do we create it and assign it to this field?

(tried to find it in the code, but couldn't.)

Reply:
this is 1 publishing http client per beacon node.

I can also have 1 publishing http client for all beacon nodes, let me know if that is better.

Reply:
its initialised [here](https://github.com/flashbots/mev-boost-relay/pull/608/files#diff-77468d40b783b541b86c07535f04917be5a5b2436ea0278c6941317a4a46f270R37)?